operator/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.13
 
 require (
 	github.com/go-logr/logr v0.4.0
-	github.com/google/uuid v1.2.0
+	github.com/google/uuid v1.2.0 // indirect
```

Comment:
is it needed?

Reply:
It works without information in comment that it is indirect dependency, `go mod tidy` adds this information when dependency is not directly used in the project. Indirect dependency, is basically dependency that is not  listed in go.mod of direct dependency, but it is still required.
I removed import [here](https://github.com/SumoLogic/tailing-sidecar/pull/160/files#diff-54a191e1f3a83568fddc0da3fc41ab5040dcbd6ac7fe022e3315345a965cbbc9L28) so now it is indirect dependency.

Reply:
Could we just removed it?